tests/realtime/redisPublisher.js

```diff
@@ -206,23 +208,39 @@ describe('tests/realtime/redisPublisher.js >', () => {
           expect(pubObj.absolutePath).to.equal(subjectNA.name);
           expect(pubObj.aspect.tags.length).to.equal(0);
           expect(pubObj.aspect.writers).to.be.undefined;
-
+          expect(pubObj.subject.absolutePath).to.be.equal(pubObj.absolutePath);
           done();
         })
         .catch(done);
       });
     });
 
+    it('Must be able to handle when Error from real time attachAspectSubject',
+    (done) => {
```

Comment:
Is this supposed to be testing the catch on line 144? If so, you would expect the promise to be resolved here, right?

Reply:
That's true @jgraff2. We were catching the exception at the upper level (bulk script) then I forgot to change the test once we move the catch to redisPublisher.

Thanks!